config/config.go

```diff
@@ -436,6 +515,38 @@ func (e *ErrConfigValidationFailed) Error() string {
 		e.UndecodedItems, ", "))
 }
 
+// ErrConfigInstanceSection error is used to warning the user that the config options in 'instance'
+// are also set in another place.
+type ErrConfigInstanceSection struct {
+	confFile           string
+	configSections     *[]InstanceConfigSection
+	deprecatedSections *[]InstanceConfigSection
+}
+
+func (e *ErrConfigInstanceSection) Error() string {
+	var builder strings.Builder
+	if len(*e.configSections) > 0 {
+		builder.WriteString("Conflict configuration options exists on both [instance] section and some other sections. ")
+	}
+	if len(*e.deprecatedSections) > 0 {
+		builder.WriteString("Some configuration options should be moved to [instance] section. ")
+	}
+	builder.WriteString("Please use the latter config options in [instance] instead: ")
+	for _, configSection := range *e.configSections {
+		for oldName, newName := range configSection.NameMappings {
+			builder.WriteString(fmt.Sprintf(" (%s, %s)", oldName, newName))
+		}
+	}
+	for _, configSection := range *e.deprecatedSections {
+		for oldName, newName := range configSection.NameMappings {
+			builder.WriteString(fmt.Sprintf(" (%s, %s)", oldName, newName))
+		}
+	}
+	builder.WriteString(". ")
```

Comment:
```suggestion
	builder.WriteString(".")
```
Please remove the space.